draft-ietf-masque-connect-udp-listen.md

```diff
@@ -164,6 +166,12 @@ and therefore is to be ignored). This document does not define any parameters
 for the connect-udp-listen header field value, but future documents might define
 parameters. Receivers MUST ignore unknown parameters.
 
+
+# The proxy-public-address Response Header Field {#response}
+Upon accepting the request, the "proxy-public-address" response header,
+MUST contain the IP and the UDP port which are allocated to the client by the
+proxy in the format {IP-Address}:{UDP-Port}
```

Comment:
We'll want to define the syntax more formally. You can use [9484](https://www.rfc-editor.org/rfc/rfc9484#name-uri-template-variable-forma) as an example

Reply:
Done, lmk what you think of the new definition